Makefile

```
@@ -253,11 +253,17 @@ bin/save-fuel-price-data: .server_generate.stamp
 bin_linux/save-fuel-price-data: .server_generate_linux.stamp
 	GOOS=linux GOARCH=amd64 go build -ldflags "$(LDFLAGS)" -o bin_linux/save-fuel-price-data ./cmd/save_fuel_price_data
 
+bin/send-post-move-survey-email: .server_generate.stamp
```

Both of these need to be changed to this:

```suggestion
bin/send-post-move-survey-email: .server_generate.stamp pkg/assets/assets.go
```

---

added dependency to both targets.